ftl/common/cache.py

```diff
@@ -94,34 +94,44 @@ def _tag(self, base_image, namespace, cache_key):
             namespace=namespace,
             tag=hashlib.sha256(fingerprint).hexdigest()))
 
+    '''Attempt to retrieve value from cache.
```

Comment:
Python docstrings go inside the function:

```python
def foo():
"""Foo does this"""
```

Reply:
whoops, fixed